docs/developer-guide/syntax-reference.rst

```diff
@@ -1656,11 +1656,16 @@ Scalar functions
 |                        |                                                                           | will return ``My Test -nnn``.                     |
 +------------------------+---------------------------------------------------------------------------+---------------------------------------------------+
 | RANDOM                 |  ``RANDOM()``                                                             | Return a random DOUBLE value between 0.0 and 1.0. |
-+------------------------+---------------------------------------------------------------------------+---------------------------------------------------+
++------------------------+---------------------------------------------------------------------------+---------------------------------------------------|
 | REPLACE                |  ``REPLACE(col1, 'foo', 'bar')``                                          | Replace all instances of a substring in a string  |
 |                        |                                                                           | with a new string.                                |
-+------------------------+---------------------------------------------------------------------------+---------------------------------------------------+
-| ROUND                  |  ``ROUND(col1)``                                                          | Round a value to the nearest BIGINT value.        |
++------------------------+---------------------------------------------------------------------------+---------------------------------------------------|
+| ROUND                  |  ``ROUND(col1)`` or ``ROUND(col1, scale)``                                | Round a value to the nearest value at the given   |
+|                        |                                                                           | number of decimal places.                         |
+|                        |                                                                           | Numbers equidistant to the nearest value will be  |
+|                        |                                                                           | rounded up (ie in a positive direction).          |
+|                        |                                                                           | If the number of decimal places is not provided   |
```

Comment:
Text talks about 'decimal places' but method signature has 'scale'.  Might be good to stick to one, or some how tie them together in the text.

Also, explain how a negative scale can be used.